src/Exporter.php

```diff
@@ -325,7 +325,7 @@ public function getProducts($itemsPerPage = null, $page = 1)
                         $products[$product['id']] = $product;
                     } else {
                         $this->trackSkippedProducts($product['id']);
-                        $this->getLog()->trace('Product was skipped as it has no id.');
+                        $this->getLog()->debug('Product was skipped as it has no id.');
```

Comment:
:memo: Monolog does not support `trace` as it is not defined in [RFC 5424](http://tools.ietf.org/html/rfc5424).

Reply:
Should be fine.